docs/success-guide/setup_config_files.md

```diff
@@ -126,25 +126,46 @@ Limits on deletion prevent accidental account deletion in the event of misconfig
+&#9744; If you need to use this feature, add lines such as below to the config file at the top level.   You can create user groups and add users to that user group on the Adobe Admin Console, and then list that user group as excluded from User Sync processing to protect that set of users.  You can list specific users and/or a pattern that matches specific user names to protect those users.  You can protect users based on their identity type as well.  For example, often User Sync is used only to manage federatedID or enterpriseID user types and you can exclude adobeID type users from management by User Sync.  You only need to include configuration items for exclusions that you wish to use.
 
-	user_removal:
-	    - exclude_group: special_users   
-	    - exclude:   ".*@example.com"   
-	    - exclude_adobe_id              
+```YAML
+dashboard_user_exclusions:
+  exclude_groups: 
+    - administrators   # Names an Adobe user group or product configuration whose members are not to be altered or removed by User Sync
+    - contractors      # You can have more than one group in a list
+  exclude_users:
+    - ".*@example.com"
+    - important_user@gmail.com
+  exclude_identity_types:
+    - adobeID          # adobeID, enterpriseID, and/or federatedID
+```
+
+
+In the above, administrators, contractors, and the user names are example values.  You would use the names of Adobe user groups, product configurations, or users that you have created.
+
+`exclude_groups` defines a list of Adobe user groups or product configurations. Adobe users who are members of such groups are neither removed, updated, nor have their group membership changed.
+
+`exclude_users` gives a list of patterns. Adobe users with usernames that match (default case-insensitive, unless the pattern specifies case-sensitive) any of the specified patterns (if there are more than one) are neither removed, updated, nor have their group membership changed.
+
+`exclude_identity_types` gives a list of identity types. Adobe users who have one of these identity types are neither removed, updated, nor have their group membership changed.
+
+Note that:
+
+- You generally wouldn't use all three exclude options.
+
+- Directory side users are still created on the Adobe side even if one of the exclude parameters would then exclude the user on the Adobe side from updates in successive runs. That is, these parameters apply only to Adobe users who exist when the Adobe directory is being matched against the customer directory.
+
+- The entire `dashboard_user_exclusions` configuration section is optional. 
+
+- Accounts that would have been removed or updated but were not because of this feature are listed as `info` level log entries.
+
+- Federated accounts that are not in the enterprise directory or are disabled in the enterprise directory cannot log in anyway (because login is handled by the ID provider and the user is no longer listed there) even if the account still exists in the Adobe dashboard.
```

Comment:
I ruthlessly avoid using the term "customer" in customer documentation.  We use the term "directory" very consistently throughout and it is defined in the Terminology section in getting started so I think we can just use it for now.  Other changes in.